data/gmh/Jellinek_1904_FriedrichVonSchwaben/stemma.gv

```diff
@@ -0,0 +1,25 @@
+digraph {
+    X [label="*X", color="grey"]
+    Z [label="*Z", color="grey"]
+    Y [label="*Y", color="grey"]
+    Y1 [label="(*Y¹)", color="grey"] # There is no explanation of why this group is represented in parenthesis.
+    IbSD [label="", color="grey"]
+
+    I [label="Iᵃ+Iᵇ"] # Represents the two nodes Iᵃ and Iᵇ of the drawn stemma, which correspond to only one manuscript, I, written by two different scribes.
```

Comment:
Couldn't we do two nodes for `Ia` and `Ib` ? Given it is two different scribes, I guess it could hypothetically be two different units ?

Reply:
Yes, that sure is a possibility. 

This stemma got me thinking a lot about what contamination means. The editor thinks that scribe a copied until a certain part and then scribe b continued at some later point in time, but he had a different source. 

At first I thought that, as we have one continuos witness of the full work that uses two different sources, this means contamination. But maybe you're right, we should consider "contamination" to be only the mix of readings from different sources. And in this case we can say that each source is the unique source for each of the two units. It would also make the encoded stemma look like the image.

Reply:
In the new commit as two different nodes

Reply:
Well, I agree there are two different kinds of contamination, the successive (easy) kind, were several sources are used successively (many possible reason, including material lacuna in the source) and simultaneous "editing" based on using several sources (e.g., two manuscripts, or one manuscript with interlinear variants added, etc.).

Cases of the first may be a bit hard to assess because you also can have the suspicion that some owner used a quire from somewhere with another quire from somewhere else to get a full text (in which case, you have two distinct units). Codicology can help, but we might not have the information…

Reply:
And, also, since a and b are separated in time, it can also be that someone decided at some point to get the (non existant) end of his copy from somewhere else. There, we can get into a lot of theoretical discussion on whether it is two different witnesses or not.

Reply:
I'd suggest the rule that it is: editor's choice.
Are there two different sigla ? If yes, two different nodes. What do you think ? Not very satisfying, but easy enough.
OR: are there two nodes on the stemma ? (even easier).

Reply:
That sounds like a good rule of thumb 👍 

Although, I think I've seen some stemma where the editor created two nodes for the same manuscript, just because it was easier and clearer to create the drawing that way than have go across from one side to the other. But in that case, it's just a visualisation issue; in this case there is a clearer editorial statement about this manuscript being composed of two different units.

Reply:
A related issue: should we labeled it as "contamination: yes" or "no" in the metadata?

Reply:
Maybe "yes"? I wonder if the use of that field is just to be able to filter all pertinent stemmata that have some sort of contamination in a general sense? And then one would need to filter again for specific interests?

Reply:
I agree, so contamination: yes, and that's how I would conceive its usage too.
I opened an issue for us to add that to the doc next documentation update (perhaps on a separate "in-depth guidelines" page),

https://github.com/OpenStemmata/OpenStemmata.github.io/issues/9